packages/@holaplexui-react/src/components/Popover.tsx

```diff
@@ -1,15 +1,15 @@
 import { Popover, Transition } from '@headlessui/react';
 import clsx from 'clsx';
-import { Fragment } from 'react';
+import { Fragment, ReactNode } from 'react';
 
 export function PopoverBox({
   triggerButton,
   elements,
-  forceDirection,
+  children,
 }: {
   triggerButton: JSX.Element;
-  elements: JSX.Element[];
-  forceDirection?: 'left';
+  elements?: JSX.Element[];
+  children?: ReactNode;
 }) {
   return (
     <Popover className="relative">
```

Comment:
Can you look into integrating react-popper into our popover to allow for setting positioning with props?

https://headlessui.com/react/popover#positioning-the-panel